packages/langium/test/grammar/grammar-validator.test.ts

```diff
@@ -0,0 +1,34 @@
+/******************************************************************************
+ * Copyright 2022 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
```

Comment:
We already have a file for testing grammar validations at `test/validation/grammar-validation.test.ts`.